src/dbt/kipptaf/models/extracts/tableau/intermediate/int_tableau__student_enrollments.sql

```diff
@@ -1,31 +1,91 @@
+with
+    ms_grad_sub as (
+        select
+            _dbt_source_relation,
+            student_number,
+            school_abbreviation as ms_attended,
+            row_number() over (
+                partition by student_number order by exitdate desc
+            ) as rn,
+        from {{ ref("base_powerschool__student_enrollments") }}
+        where school_level = 'MS'
+    )

```

Comment:
note to self: can we move this further upstream?

Reply:
> note to self: can we move this further upstream?

Should we dump it on student_filters?

Reply:
maybe, but that's no upstream from this anyway. made an issue for myself

Reply:
Does this mean we can approve this PR with what we have for now? (pretty please!)